GitUI/GitUICommands.cs

```diff
@@ -1746,15 +1756,15 @@ private bool RunFileHistoryCommand(IReadOnlyList<string> args, bool showBlame)
                                  () => new FormBrowse(commands: this, new BrowseArguments
                                  {
                                      RevFilter = filterByRevision ? revision?.ObjectId.ToString() : null,
-                                     PathFilter = fileHistoryFileName,
+                                     PathFilter = fileHistoryFileName.QuoteNE(),
                                      SelectedId = revision?.ObjectId,
                                      IsFileBlameHistory = true
                                  }));
             }
             else
             {
                 ShowModelessForm(owner: null, requiresValidWorkingDir: true, preEvent: null, postEvent: null,
-                                 () => new FormFileHistory(commands: this, fileHistoryFileName, revision, filterByRevision, showBlame));
+                                 () => new FormFileHistory(commands: this, fileHistoryFileName.QuoteNE(), revision, filterByRevision, showBlame));
```

Comment:
Then this can be done before the `if` in one place.

Though IIRC, @gerhardol has added support for filtering for multiple files recently.
Does this still work with the extra quotes?

Reply:
This still works.
This code is used to open new GE instances. Obviously only one file can be handled with this implementation.
Multiple files in the current instance is not new, it was possible to add files in FormFilter (with or without quotes) before too. What I added was to add several files from RevDiff. This was not straightforward for multiple files and opening new instances, so that was not added.

Reply:
Since it is needed in all branches, quote the filename in one place, please - right after checking it for `IsNullOrWhiteSpace`.